website/docs/d/search_index.html.markdown

```diff
@@ -0,0 +1,35 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: search index"
+sidebar_current: "docs-mongodbatlas-search-index"
+description: |-
+Describes a Search Index.
+---
+
+# mongodbatlas_search_index
+
+`mongodbatlas_search_index` describe all Projects. This represents projects that have been created.
```

Comment:
This is likely a copy and paste mistake :)

Reply:
my bad!